runtime/polkadot/src/lib.rs

```diff
@@ -944,6 +945,46 @@ impl poll::Trait for Runtime {
 	type End = PollEnd;
 }
 
+parameter_types! {
+	pub const MaxStatementLength: usize = 1_000;
+	pub const UnlockedProportion: Permill = Permill::zero();
+	pub const MaxUnlocked: u64 = 0;
```

Comment:
```suggestion
	pub const MaxUnlocked: Balance = Zero::zero();
```

This should be the balance type... how is this being allowed?